charts/pega/README.md

```diff
@@ -137,6 +137,24 @@ docker:
     imagePullPolicy: "Always"
 ```
 
+## Customize Utility Image Locations
+The Pega chart, installer chart, and pegasearch charts use the busybox and dcasavant/k8s-wait-for images for various utility tasks.  
+You can configure where and how these images are pulled by modifying `utilityImages.busybox` and `utilityImages.k8s_wait_for`.
```

Comment:
can we try to take a higher level view for this??
## Pega-provided utilities image
Pega deployment charts include two images, `busybox` and `dcasavant/k8s-wait-for` Dockers images, that you can add to your deployment to complete various tasks in your clusters. these utilities include:
>>>can you give us a list or state a business value??<<<
 To add them to your deployment, you configure the image name and images pull policy by modifying `utilityImages.busybox` and `utilityImages.k8s_wait_for` parameters as described below.

Reply:
If you are including this in the SRS readme, then I think that readme needs the same introduction or put the main introduction in the main readme so you don't have to repeat content>>https://github.com/pegasystems/pega-helm-charts/blob/master/README.md and then link appropriately to the readmes from the main readme.

Reply:
Since we're not "adding" them to the deployment, and they're required, I might try another stab at it.  The business value they're adding is being able to correctly deploy the charts, essentially.

## Customize Utility Images Locations
The Pega and Installer charts require the `busybox` and `dcasavant/k8s-wait-for` images to deploy successfully.  Since some clients may not want to use the default DockerHub registry to pull these images from, we let customers configure where these images are pulled from by exposing `utilityImages.busybox` and `utilityImages.k8s_wait_for`.  These fields must have a value.

Reply:
I'm between section headings.  
- Busybox and k8s-wait-for utility images
- Utility images
- Pega-provided utility images

Reply:
how about>>

Deploying with Pega-provided Busybox and k8s-wait-for utility images
To deploy Pega Platform, the Pega helm  chart requires the use of the busybox and dcasavant/k8s-wait-for images. For clients who want to pull these images from non-default DockerHub registry they must push and tag these images to a private repository, and then pull these images by specifying `busybox.image` and `busybox.imagePullPolicy` values as described below.

Reply:
Here's the above with a few slight changes to make it more technically accurate.  The one thing I wasn't 100% on was the end of the last line.  They have to modify `k8s-wait-for.image` and `k8s-wait-for.imagePullPolicy` as well as `busybox.image` and `busybox.imagePullPolicy`, so to make it more succinct is why I was kept it at `busybox` and `k8s-wait-for`

## Deploying with Pega-provided busybox and k8s-wait-for utility images
To deploy Pega Platform, the Pega helm chart requires the use of the busybox and k8s-wait-for images.  For clients who want to pull these images from a registry other than Docker Hub, they must tag and push these images to another registry, and then pull these images by specifying `busybox` and `k8s-wait-for` values as described below.

Reply:
looks good! When you have the pega and SRS charts updated, I can do a final review there! 

Thanks for the excellent collaboration!

Reply:
Ok changes have been made!  Thanks for the help Taz!

Reply:
sure thing!